constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -205,114 +205,24 @@ public boolean verify() throws ConstructorException {
-     * @return true if working
-     * @throws ConstructorException if the request is invalid.
-     */
-    public boolean addItem(ConstructorItem item, String autocompleteSection) throws ConstructorException {
-        try {
-            HttpUrl url = this.makeUrl(Arrays.asList("v1", "item"));
-            Map<String, Object> data = item.toMap();
-            data.put("autocomplete_section", autocompleteSection);
-            String params = new Gson().toJson(data);
-            RequestBody body = RequestBody.create(MediaType.parse("application/json; charset=utf-8"), params);
-            Request request = this.makeAuthorizedRequestBuilder()
-                .url(url)
-                .post(body)
-                .build();
-
-            Response response = client.newCall(request).execute();
-            getResponseBody(response);
-            return true;
-        } catch (Exception exception) {
-            throw new ConstructorException(exception);
-        }
-    }
-
-    /**
-     * Adds an item to your autocomplete or updates it if it already exists.
-     *
-     * @param item the item that you're adding.
-     * @param autocompleteSection the section of the autocomplete that you're adding the item to.
-     * @return true if working
-     * @throws ConstructorException if the request is invalid.
-     */
-    public boolean addOrUpdateItem(ConstructorItem item, String autocompleteSection) throws ConstructorException {
-        try {
-            HttpUrl url = this.makeUrl(Arrays.asList("v1", "item"));
-            url = url.newBuilder().addQueryParameter("force", "1").build();
-            Map<String, Object> data = item.toMap();
-            data.put("autocomplete_section", autocompleteSection);
-            String params = new Gson().toJson(data);
-            RequestBody body = RequestBody.create(MediaType.parse("application/json; charset=utf-8"), params);
-            Request request = this.makeAuthorizedRequestBuilder()
-                .url(url)
-                .put(body)
-                .build();
-
-            Response response = client.newCall(request).execute();
-            getResponseBody(response);
-            return true;
-        } catch (Exception exception) {
-            throw new ConstructorException(exception);
-        }
-    }
-
-    /**
-     * Adds multiple items to your autocomplete (limit of 1000 items)
+     * Adds multiple items to your index whilst updating existing ones (limit of 1000 items)
      *
      * @param items the items you want to add.
      * @param autocompleteSection the section of the autocomplete that you're adding the items to.
      * @return true if working
      * @throws ConstructorException if the request is invalid.
      */
-    public boolean addItemBatch(ConstructorItem[] items, String autocompleteSection) throws ConstructorException {
+    public boolean addOrUpdateItems(ConstructorItem[] items, String autocompleteSection) throws ConstructorException {
```

Comment:
Can we change `autocompleteSection` to `section` or will that lead to inconsistencies across the repo?